core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1001,7 +1001,13 @@ class KafkaApis(val requestChannel: RequestChannel,
 
   def handleApiVersionRequest(request: RequestChannel.Request) {
     val responseHeader = new ResponseHeader(request.header.correlationId)
-    val responseBody = KafkaApis.apiVersionResponse
+    def isApiVersionRequestVersionSupported: Boolean = {
```

Comment:
I think you want "val" and not "def" here, since this is not a method.
and make sure the style matches our conventions